apps/designer/app/designer/features/style-panel/controls/toggle/toggle-control.tsx

```diff
@@ -0,0 +1,41 @@
+import { Flex, ToggleGroup, Tooltip } from "@webstudio-is/design-system";
+import { StyleProperty } from "@webstudio-is/react-sdk";
+import { useIsFromCurrentBreakpoint } from "../../shared/use-is-from-current-breakpoint";
+
+export type ToggleGroupControlProps = {
+  property: StyleProperty | StyleProperty[];
```

Comment:
is there a concrete use case of passing an array of properties? If not we should simplify and use single property.